ios/PurchasesHybridCommon/PurchasesHybridCommon/PurchaseParams+HybridAdditions.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  PurchaseParams+HybridAdditions.swift
+//  PurchasesHybridCommon
+//
+//  Created by Will Taylor on 1/7/25.
+//  Copyright © 2025 RevenueCat. All rights reserved.
+//
+
+import Foundation
+import RevenueCat
+
+@objc
+public extension PurchaseParams.Builder {
+
+    // For some reason, PurchaseParams.Builder isn't exposed as a part of the PHC's public API to KMP by default.
```

Comment:
I think it makes sense it's not exposed. PHC doesn't have any public method that either takes `PurchaseParams.Builder` as an argument or returns it, so the type is not part of it's public API. (Or am I missing something?)

If what I'm saying makes sense, maybe we can remove "For some reason" here? 
```suggestion
    // PurchaseParams.Builder isn't exposed as a part of the PHC's public API to KMP by default.
```